src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/MapFromPortal/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Name": "Add Map from exsiting Portal Item",
+  "SampleName": "MapFromPortal",
+  "Description": "This sample illustrates how to open an existing web map from a portal.",
```

Comment:
Could be something like : This sample demonstrates how to open an existing web map from a portal. The sample opens with a web map displayed by default. You can change the shown web map by selecting a new  one from the populated list or provide web map id.

Remember to cross reference sample descriptions and content with other team ones.